mne/inverse_sparse/mxne_optim.py

```diff
@@ -714,26 +714,141 @@ def norm_l1_tf(Z, shape, n_orient):
     return l1_norm
 
 
-@verbose
-def _tf_mixed_norm_solver_bcd_(M, G, Z, active_set, alpha_space, alpha_time,
-                               lipschitz_constant, phi, phiT,
-                               wsize=64, tstep=4, n_orient=1,
-                               maxit=200, tol=1e-8, log_objective=True,
-                               perc=None, verbose=None):
+def prox_l1_full(Y, alpha, n_orient):
```

Comment:
We called it prox_l1 also in the paper, but with a description that it is a block thresholding if free or loose source orientations are used. 
@mathurinm would adding such a description be enough?
@agramfort what's your opinion on changing the function name?